src/image-sync-manager.ts

```diff
@@ -0,0 +1,46 @@
+import {ImagePlayer} from './image-player';
+
+export class ImageSyncManager {
+  _eventManager: KalturaPlayerTypes.EventManager;
+  _mainPlayer: KalturaPlayerTypes.Player;
+  _imagePlayer: ImagePlayer;
+  _logger: KalturaPlayerTypes.Logger;
+
+  constructor(
+    eventManager: KalturaPlayerTypes.EventManager,
+    mainPlayer: KalturaPlayerTypes.Player,
+    imagePlayer: ImagePlayer,
+    logger: KalturaPlayerTypes.Logger
+  ) {
+    this._eventManager = eventManager;
+    this._mainPlayer = mainPlayer;
+    this._imagePlayer = imagePlayer;
+    this._logger = logger;
+    this._syncEvents();
+  }
+
+  private _syncEvents = () => {
+    this._eventManager.listen(this._mainPlayer, this._mainPlayer.Event.TIMED_METADATA, this._onTimedMetadata);
+    this._eventManager.listen(this._mainPlayer, 'timed_metadata_added', this._onTimedMetadataAdded); // TODO: should be enum from cuePointManager
```

Comment:
the convention is `timedmetadataadded` also in dispatch (https://github.com/kaltura/playkit-js-dual-screen/blob/e4572a00aae0c70efe20e871216b570628808bbe/src/dualscreen.tsx#L458)